integration/tests/posit/connect/test_users.py

```diff
@@ -29,8 +33,8 @@ def test_lock(self):
         assert len(self.client.users.find(account_status="locked")) == 0
 
     def test_count(self):
-        # aron, bill, cole, and me
-        assert self.client.users.count() == 4
+        # aron, bill, cole, and me (and existing user)
+        assert self.client.users.count() == 3 + self.existing_users
```

Comment:
Could you say more about why `3 + self.existing_users` is preferred to the hard code?

It feels a little off that this blends something from setup with a test here: if `count()` were to not work at all, that would make the setup always fail. And if we wanted to test that `client.users.count(); client.users.create(); client.users.count()` increments like we expect, that should be in a test block, right?

Reply:
In the test file above (integration/tests/posit/connect/test_content_item_permissions.py), I want to use a couple users to test the permissions. So I added them there to have it be self contained. (We could move the users to an earlier location, but enforcing that the users exist for any subset of testing seemed like a big hassle.)

In slack, Aron stated that users can not be deleted from within Connect. So if another user is added in a different test, then the total number of users in this test will be larger than 4 (aron, bill, cole, and me). 

> It feels a little off that this blends something from setup with a test here

Agreed. But each of the other tests require the three user objects to already exist. 

We could move the `.create()` test to another test class and retrieve the users in this class to avoid crossing setup / testing. Using a subset of tests won't work well if trying to just test this situation but that's ok.

Reply:
> In slack, Aron stated that users can not be deleted from within Connect. So if another user is added in a different test, then the total number of users in this test will be larger than 4 (aron, bill, cole, and me).


_nods_ right right, that's true, but also true if we have `4` there or if we do `3 + self.existing_users`, right? We would need to do `4 + self.existing_users` or `5 + self.existing_users` if we add more users. Or am I missing something?

Reply:
In this test we only add 3x users: `aron`, `bill`, and `cole`. So only if we add more users within _this_ test will we need to change `3` to a new value.